<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>415</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>515</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>515</width>
    <height>415</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbPlay">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbStop">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuColor">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <widget class="QMenu" name="menuChekers">
      <property name="title">
       <string>Chekers</string>
      </property>
      <widget class="QMenu" name="menuReds">
       <property name="title">
        <string>Reds</string>
       </property>
       <addaction name="action1"/>
       <addaction name="action2"/>
       <addaction name="action3"/>
      </widget>
      <widget class="QMenu" name="menuBLues">
       <property name="title">
        <string>Blues</string>
       </property>
       <addaction name="action1_2"/>
       <addaction name="action2_2"/>
       <addaction name="action3_2"/>
      </widget>
      <addaction name="menuReds"/>
      <addaction name="menuBLues"/>
     </widget>
     <widget class="QMenu" name="menuBackground">
      <property name="title">
       <string>Background</string>
      </property>
      <addaction name="action1_3"/>
      <addaction name="action2_3"/>
      <addaction name="action3_3"/>
     </widget>
     <addaction name="menuChekers"/>
     <addaction name="menuBackground"/>
     <addaction name="actionDefault"/>
    </widget>
    <addaction name="menuColor"/>
   </widget>
   <widget class="QMenu" name="menuMore_app">
    <property name="title">
     <string>More app</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionApp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuMore_app"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionRules_for_game">
   <property name="text">
    <string>Rules for game</string>
   </property>
  </action>
  <action name="actionAbut_programm">
   <property name="text">
    <string>About programm</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionApp">
   <property name="text">
    <string>App</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
